public/app/features/dashboard-scene/scene/setDashboardPanelContext.ts

```diff
@@ -160,6 +161,7 @@ export function getAdHocFilterVariableFor(scene: DashboardScene, ds: DataSourceR
   const newVariable = new AdHocFiltersVariable({
     name: 'Filters',
     datasource: ds,
+    supportsMultiValueOperators: config.featureToggles.adhocFilterOneOf,
```

Comment:
won't this depend on the datasource?

Reply:
@torkelo - @ashharrison90 plans to add support for this new operator to all core data sources supporting filters.

Reply:
sounds like a lot of work, and what about all external & enterprise ones ?

Reply:
ah i was working under the assumption that only prometheus/loki/influx/elastic supported adhoc filters. from our docs:

![image](https://github.com/user-attachments/assets/d833e4ad-50c0-4fee-aaad-0045857a3aad)

if that's not the case and there's more, then that might not be feasible 😬

Reply:
yeah, there's also this, probably not comprehensive list of datasources using filters api: https://github.com/search?q=grafana+getTagKeys+language%3ATypeScript&type=code

Reply:
@torkelo  / @ashharrison90  WDYT about putting this config into plugin.json, this way we would not have to wait for the plugin to load in order to know whether or not the plugin supports multi value filters or not.

Reply:
@ashharrison90 that is a strange doc note, its very far from true, in my research many more data sources support it, at least 30-50% of our enterprise data sources support it in some form

Reply:
no idea what percentage of external / community ones, but clickhouse does and think many more

Reply:
got it - ok adding support to all of them is probably not feasible then

@dprokop that sounds fine to me, but i have to admit i don't know exactly what the full flow would look like there. would we need to speak to plugins platform to see if that makes sense with them as well? 🤔

Reply:
@ashharrison90 the entire thing boils down to extending [`DataSourcePluginMeta`](https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/datasource.ts#L129) with a new boolean. The plugin metadata becomes available via `config.datasources[ds-plugin-type]` so we could actually read it directly in `AdHocFiltersVariable` constructor set `supportsMultiValueOperators` accordingly.

As you siggested, let's bring this up with @grafana/plugins-platform if there are any gotchas with this approach.

Reply:
i think it would be a possible way forward. The only drawback that I can see is that we introduce yet another value in `plugin.json` that risk of being out of sync with what the plugin actually support. A plugin can define this value as true but don't support it in the code base and the other way around. With that said we already have this on other places. 

We are looking in to other patterns to be able to generate the static information during build time so it might not be an issue in the future.

Reply:
> We are looking in to other patterns to be able to generate the static information during build time so it might not be an issue in the future.

@mckn - do you consider this an acceptable approach until another pattern is established?

Reply:
Yeah, if it help you guys moving forward I think it is fine 👍🏻  Please ping the @grafana/plugins-platform when you have a suggestion on what that property should be named and we will put a ✅  on the PR.

Reply:
have ended up naming it `multiValueFilterOperators` - please let me know your thoughts on this 🙏 (naming is hard!)

cc @grafana/plugins-platform

Reply:
LGTM